packages/client/src/generation/TSClient/PrismaClient.ts

```diff
@@ -307,6 +307,22 @@ function runCommandRawDefinition(this: PrismaClientClass) {
   return ts.stringify(method, { indentLevel: 1, newLine: 'leading' })
 }
 
+function applyPendingMigrationsDefinition(this: PrismaClientClass) {
+  const method = ts.method('$applyPendingMigrations').setReturnType(ts.prismaPromise(ts.voidType))
```

Comment:
In case we want to keep explicit method:

```suggestion
  const method = ts.method('$applyPendingMigrations').setReturnType(ts.promise(ts.voidType))
```

`PrismaPromise` is a special thing that among other things means it can be passed to `$transaction`, which is not the case for `applyPendingMigrations`